Makefile

```diff
@@ -54,6 +66,35 @@ build-image-ocp-example: build-base-image ## Build a rag-content container image
 build-base-image: ## Build base container image
 	podman build -t $(TORCH_GROUP)-road-core-base -f Containerfile.base --build-arg FLAVOR=$(TORCH_GROUP)
 
+start-postgres:
+	podman run -d --name pgvector --rm -e POSTGRES_PASSWORD=$(POSTGRES_PASSWORD) \
+	 -p $(POSTGRES_PORT):5432 \
+	 -v $(PWD)/postgresql/data:/var/lib/postgresql/data:Z ankane/pgvector
```

Comment:
If I'm reading this right and this image comes from Docker Hub, we can get hit with upcoming pull rate limits: https://docs.docker.com/docker-hub/usage/pulls. Any chance we can use https://catalog.redhat.com/software/containers/rhel9/postgresql-16/657b03866783e1b1fb87e142 ?

Reply:
I think it should work as long as the pgvector extension is installed, but I do not know how to install it on the top of rhel9/postgresql image... 

I used ankane/pgvector here since it contains pgvector.  [The docker hub page of ankane/pgvector](https://hub.docker.com/r/ankane/pgvector) states:

> For pgvector 0.6.0+, use [pgvector/pgvector](https://hub.docker.com/r/pgvector/pgvector) instead.

so maybe I should use pgvector/pgvector here. 

According to [Dockerfile for pgvector/pgvector](https://github.com/pgvector/pgvector/blob/master/Dockerfile), it is built from offcial Postgresql image, which is build on a Ubuntu image.

Reply:
I found [Fedora rpms of pgvector for Postgres 16](https://src.fedoraproject.org/rpms/postgresql16-pgvector). So if we want to run pgvector on RHEL-based Linux, we should be able to build using a base image like ubi9/ubi-minimal, we can install pgvetor using dnf.

This Makefile target is provided as an example of running a Postgres instance. I think using the pgvector/pgvector image would be sufficient. What do you think?

Reply:
@syedriko Is the use of the pgvector/pgvector:pg16 image acceptable to you?